client/components/overview/StyleButton.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { ButtonStyle } from './styles/StyleSelector.style';
 
-const StyleButton = ({ style }) => (
-  <input type="image" name="style-button" src={style.photos.thumbnail_url} border="Specfiy Image Border " alt="style" />
```

Comment:
Your section is looking cool after adding these styles. 
Just a minor comment, consider putting each attribute in a separate line, but no need to update before the merge. I found myself put everything on one line all the time, haha.

Reply:
Thank you for catching this!! I'm going to make those changes for code cleanliness, then merge!